### ISSUE TITLE
2025-03-05, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#102]
# Today's Arxiv Papers

(1) [Wikipedia in the Era of LLMs: Evolution and Risks](http://arxiv.org/abs/2503.02879v1), 2025-03-04

(2) [Language Models can Self-Improve at State-Value Estimation for Better Search](http://arxiv.org/abs/2503.02878v1), 2025-03-04

(3) [The First Few Tokens Are All You Need: An Efficient and Effective Unsupervised Prefix Fine-Tuning Method for Reasoning Models](http://dx.doi.org/10.13140/RG.2.2.33772.07043), 2025-03-04

(4) [Prompting Generative AI with Interaction-Augmented Instructions](http://arxiv.org/abs/2503.02874v1), 2025-03-04

(5) [FairSense-AI: Responsible AI Meets Sustainability](http://arxiv.org/abs/2503.02865v1), 2025-03-04

(6) [Calibrating LLM Confidence with Semantic Steering: A Multi-Prompt Aggregation Framework](http://arxiv.org/abs/2503.02863v1), 2025-03-04

(7) [Privacy and Accuracy-Aware AI/ML Model Deduplication](http://arxiv.org/abs/2503.02862v1), 2025-03-04

(8) [Shakespearean Sparks: The Dance of Hallucination and Creativity in LLMs' Decoding Layers](http://arxiv.org/abs/2503.02851v1), 2025-03-04

(9) [Mask-DPO: Generalizable Fine-grained Factuality Alignment of LLMs](http://arxiv.org/abs/2503.02846v1), 2025-03-04

(10) [AlignDistil: Token-Level Language Model Alignment as Adaptive Policy Distillation](http://arxiv.org/abs/2503.02832v1), 2025-03-04

(11) [Q-Filters: Leveraging QK Geometry for Efficient KV Cache Compression](http://arxiv.org/abs/2503.02812v1), 2025-03-04

(12) [RAAD-LLM: Adaptive Anomaly Detection Using LLMs and RAG Integration](http://arxiv.org/abs/2503.02800v1), 2025-03-04

(13) [IterPref: Focal Preference Learning for Code Generation via Iterative Debugging](http://arxiv.org/abs/2503.02783v1), 2025-03-04

(14) [Multimodal AI predicts clinical outcomes of drug combinations from preclinical data](http://arxiv.org/abs/2503.02781v1), 2025-03-04

(15) [Implicit Bias in LLMs: A Survey](http://arxiv.org/abs/2503.02776v1), 2025-03-04

(16) [InSerter: Speech Instruction Following with Unsupervised Interleaved Pre-training](http://arxiv.org/abs/2503.02769v1), 2025-03-04

(17) [From Metaphor to Mechanism: How LLMs Decode Traditional Chinese Medicine Symbolic Language for Modern Clinical Relevance](http://arxiv.org/abs/2503.02760v1), 2025-03-04

(18) [BatchGEMBA: Token-Efficient Machine Translation Evaluation with Batched Prompting and Prompt Compression](http://arxiv.org/abs/2503.02756v1), 2025-03-04

(19) [Large Language Models for Multilingual Previously Fact-Checked Claim Detection](http://arxiv.org/abs/2503.02737v1), 2025-03-04

(20) [Evaluating Knowledge Generation and Self-Refinement Strategies for LLM-based Column Type Annotation](http://arxiv.org/abs/2503.02718v1), 2025-03-04

(21) [RedChronos: A Large Language Model-Based Log Analysis System for Insider Threat Detection in Enterprises](http://arxiv.org/abs/2503.02702v1), 2025-03-04

(22) [MindBridge: Scalable and Cross-Model Knowledge Editing via Memory-Augmented Modality](http://arxiv.org/abs/2503.02701v1), 2025-03-04

(23) [FlowPlan: Zero-Shot Task Planning with LLM Flow Engineering for Robotic Instruction Following](http://arxiv.org/abs/2503.02698v1), 2025-03-04

(24) [FinArena: A Human-Agent Collaboration Framework for Financial Market Analysis and Forecasting](http://arxiv.org/abs/2503.02692v1), 2025-03-04

(25) [MPO: Boosting LLM Agents with Meta Plan Optimization](http://arxiv.org/abs/2503.02682v1), 2025-03-04

(26) [Multidimensional Consistency Improves Reasoning in Language Models](http://arxiv.org/abs/2503.02670v1), 2025-03-04

(27) [LoRA-Null: Low-Rank Adaptation via Null Space for Large Language Models](http://arxiv.org/abs/2503.02659v1), 2025-03-04

(28) [The Effectiveness of Large Language Models in Transforming Unstructured Text to Standardized Formats](http://arxiv.org/abs/2503.02650v1), 2025-03-04

(29) [Reflection on Data Storytelling Tools in the Generative AI Era from the Human-AI Collaboration Perspective](http://arxiv.org/abs/2503.02631v1), 2025-03-04

(30) [Towards Event Extraction with Massive Types: LLM-based Collaborative Annotation and Partitioning Extraction](http://arxiv.org/abs/2503.02628v1), 2025-03-04

(31) [Rewarding Doubt: A Reinforcement Learning Approach to Confidence Calibration of Large Language Models](http://arxiv.org/abs/2503.02623v1), 2025-03-04

(32) [OkraLong: A Flexible Retrieval-Augmented Framework for Long-Text Query Processing](http://arxiv.org/abs/2503.02603v1), 2025-03-04

(33) [Seeing is Understanding: Unlocking Causal Attention into Modality-Mutual Attention for Multimodal LLMs](http://arxiv.org/abs/2503.02597v1), 2025-03-04

(34) [StageDesigner: Artistic Stage Generation for Scenography via Theater Scripts](http://arxiv.org/abs/2503.02595v1), 2025-03-04

(35) [MciteBench: A Benchmark for Multimodal Citation Text Generation in MLLMs](http://arxiv.org/abs/2503.02589v1), 2025-03-04

(36) [Playing games with Large language models: Randomness and strategy](http://arxiv.org/abs/2503.02582v1), 2025-03-04

(37) [LLM-Safety Evaluations Lack Robustness](http://arxiv.org/abs/2503.02574v1), 2025-03-04

(38) [SpecInF: Exploiting Idle GPU Resources in Distributed DL Training via Speculative Inference Filling](http://arxiv.org/abs/2503.02550v1), 2025-03-04

(39) [Use Me Wisely: AI-Driven Assessment for LLM Prompting Skills Development](http://arxiv.org/abs/2503.02532v1), 2025-03-04

(40) [Generator-Assistant Stepwise Rollback Framework for Large Language Model Agent](http://arxiv.org/abs/2503.02519v1), 2025-03-04

(41) [LADM: Long-context Training Data Selection with Attention-based Dependency Measurement for LLMs](http://arxiv.org/abs/2503.02502v1), 2025-03-04

(42) [PennyLang: Pioneering LLM-Based Quantum Code Generation with a Novel PennyLane-Centric Dataset](http://arxiv.org/abs/2503.02497v1), 2025-03-04

(43) [BioD2C: A Dual-level Semantic Consistency Constraint Framework for Biomedical VQA](http://arxiv.org/abs/2503.02476v1), 2025-03-04

(44) [UAV-VLRR: Vision-Language Informed NMPC for Rapid Response in UAV Search and Rescue](http://arxiv.org/abs/2503.02465v1), 2025-03-04

(45) [It Helps to Take a Second Opinion: Teaching Smaller LLMs to Deliberate Mutually via Selective Rationale Optimisation](http://arxiv.org/abs/2503.02463v1), 2025-03-04

(46) [Don't Get Too Excited -- Eliciting Emotions in LLMs](http://dx.doi.org/10.1145/3706599.3720191), 2025-03-04

(47) [Measuring What Makes You Unique: Difference-Aware User Modeling for Enhancing LLM Personalization](http://arxiv.org/abs/2503.02450v1), 2025-03-04

(48) [BRIDGE: Bootstrapping Text to Control Time-Series Generation via Multi-Agent Iterative Optimization and Diffusion Modelling](http://arxiv.org/abs/2503.02445v1), 2025-03-04

(49) [AILS-NTUA at SemEval-2025 Task 4: Parameter-Efficient Unlearning for Large Language Models using Data Chunking](http://arxiv.org/abs/2503.02443v1), 2025-03-04

(50) [AILS-NTUA at SemEval-2025 Task 3: Leveraging Large Language Models and Translation Strategies for Multilingual Hallucination Detection](http://arxiv.org/abs/2503.02442v1), 2025-03-04

(51) [NLI4DB: A Systematic Review of Natural Language Interfaces for Databases](http://arxiv.org/abs/2503.02435v1), 2025-03-04

(52) [Hierarchical Re-ranker Retriever (HRR)](http://arxiv.org/abs/2503.02401v1), 2025-03-04

(53) [Promptware Engineering: Software Engineering for LLM Prompt Development](http://arxiv.org/abs/2503.02400v1), 2025-03-04

(54) [PersonaX: A Recommendation Agent Oriented User Modeling Framework for Long Behavior Sequence](http://arxiv.org/abs/2503.02398v1), 2025-03-04

(55) [ReSo: A Reward-driven Self-organizing LLM-based Multi-Agent System for Reasoning Tasks](http://arxiv.org/abs/2503.02390v1), 2025-03-04

(56) [An Efficient and Precise Training Data Construction Framework for Process-supervised Reward Model in Mathematical Reasoning](http://arxiv.org/abs/2503.02382v1), 2025-03-04

(57) [Teaching Metric Distance to Autoregressive Multimodal Foundational Models](http://arxiv.org/abs/2503.02379v1), 2025-03-04

(58) [MedEthicEval: Evaluating Large Language Models Based on Chinese Medical Ethics](http://arxiv.org/abs/2503.02374v1), 2025-03-04

(59) [EchoQA: A Large Collection of Instruction Tuning Data for Echocardiogram Reports](http://arxiv.org/abs/2503.02365v1), 2025-03-04

(60) [Add-One-In: Incremental Sample Selection for Large Language Models via a Choice-Based Greedy Paradigm](http://arxiv.org/abs/2503.02359v1), 2025-03-04

(61) [Efficient Long Context Fine-tuning with Chunk Flow](http://arxiv.org/abs/2503.02356v1), 2025-03-04

(62) [CoServe: Efficient Collaboration-of-Experts (CoE) Model Inference with Limited Memory](http://dx.doi.org/10.1145/3676641.3715986), 2025-03-04

(63) [DeLTa: A Decoding Strategy based on Logit Trajectory Prediction Improves Factuality and Reasoning Ability](http://arxiv.org/abs/2503.02343v1), 2025-03-04

(64) [Unlocking a New Rust Programming Experience: Fast and Slow Thinking with LLMs to Conquer Undefined Behaviors](http://arxiv.org/abs/2503.02335v1), 2025-03-04

(65) [Limited Effectiveness of LLM-based Data Augmentation for COVID-19 Misinformation Stance Detection](http://dx.doi.org/10.1145/3701716.3715521), 2025-03-04

(66) [PromptCoT: Synthesizing Olympiad-level Problems for Mathematical Reasoning in Large Language Models](http://arxiv.org/abs/2503.02324v1), 2025-03-04

(67) [A Token-level Text Image Foundation Model for Document Understanding](http://arxiv.org/abs/2503.02304v1), 2025-03-04

(68) [Towards Large Language Model Guided Kernel Direct Fuzzing](http://arxiv.org/abs/2503.02301v1), 2025-03-04

(69) [Towards Explainable Doctor Recommendation with Large Language Models](http://arxiv.org/abs/2503.02298v1), 2025-03-04

(70) [Memorize or Generalize? Evaluating LLM Code Generation with Evolved Questions](http://arxiv.org/abs/2503.02296v1), 2025-03-04

(71) [AppAgentX: Evolving GUI Agents as Proficient Smartphone Users](http://arxiv.org/abs/2503.02268v1), 2025-03-04

(72) [Large Language Models as Natural Selector for Embodied Soft Robot Design](http://arxiv.org/abs/2503.02249v1), 2025-03-04

(73) [Making Better Mistakes in CLIP-Based Zero-Shot Classification with Hierarchy-Aware Language Prompts](http://arxiv.org/abs/2503.02248v1), 2025-03-04

(74) [From Code to Courtroom: LLMs as the New Software Judges](http://arxiv.org/abs/2503.02246v1), 2025-03-04

(75) [OmniSQL: Synthesizing High-quality Text-to-SQL Data at Scale](http://arxiv.org/abs/2503.02240v1), 2025-03-04

(76) [V2X-LLM: Enhancing V2X Integration and Understanding in Connected Vehicle Corridors](http://arxiv.org/abs/2503.02239v1), 2025-03-04

(77) [Haste Makes Waste: Evaluating Planning Abilities of LLMs for Efficient and Feasible Multitasking with Time Constraints Between Actions](http://arxiv.org/abs/2503.02238v1), 2025-03-04

(78) [VQ-LLM: High-performance Code Generation for Vector Quantization Augmented LLM Inference](http://arxiv.org/abs/2503.02236v1), 2025-03-04

(79) [Enhancing LLM Reliability via Explicit Knowledge Boundary Modeling](http://arxiv.org/abs/2503.02233v1), 2025-03-04

(80) [ATLaS: Agent Tuning via Learning Critical Steps](http://arxiv.org/abs/2503.02197v1), 2025-03-04

(81) [Understanding and Predicting Derailment in Toxic Conversations on GitHub](http://arxiv.org/abs/2503.02191v1), 2025-03-04

(82) [DivPrune: Diversity-based Visual Token Pruning for Large Multimodal Models](http://arxiv.org/abs/2503.02175v1), 2025-03-04

(83) [Adversarial Tokenization](http://arxiv.org/abs/2503.02174v1), 2025-03-04

(84) [Leveraging Large Language Models for Enhanced Digital Twin Modeling: Trends, Methods, and Challenges](http://arxiv.org/abs/2503.02167v1), 2025-03-04

(85) [LLM-TabFlow: Synthetic Tabular Data Generation with Inter-column Logical Relationship Preservation](http://arxiv.org/abs/2503.02161v1), 2025-03-04

(86) [Tabby: Tabular Data Synthesis with Language Models](http://arxiv.org/abs/2503.02152v1), 2025-03-04

(87) [Malware Classification from Memory Dumps Using Machine Learning, Transformers, and Large Language Models](http://arxiv.org/abs/2503.02144v1), 2025-03-04

(88) [Measuring Intrinsic Dimension of Token Embeddings](http://arxiv.org/abs/2503.02142v1), 2025-03-04

(89) [Network Traffic Classification Using Machine Learning, Transformer, and Large Language Models](http://arxiv.org/abs/2503.02141v1), 2025-03-04

(90) [TMIQ: Quantifying Test and Measurement Domain Intelligence in Large Language Models](http://arxiv.org/abs/2503.02123v1), 2025-03-03

(91) [OVAMOS: A Framework for Open-Vocabulary Multi-Object Search in Unknown Environments](http://arxiv.org/abs/2503.02106v1), 2025-03-03

(92) [Biomedical Foundation Model: A Survey](http://arxiv.org/abs/2503.02104v1), 2025-03-03

(93) [LLMs as Educational Analysts: Transforming Multimodal Data Traces into Actionable Reading Assessment Reports](http://arxiv.org/abs/2503.02099v1), 2025-03-03

(94) [Which Code Statements Implement Privacy Behaviors in Android Applications?](http://arxiv.org/abs/2503.02091v1), 2025-03-03

(95) [Linear Representations of Political Perspective Emerge in Large Language Models](http://arxiv.org/abs/2503.02080v1), 2025-03-03

(96) [Superscopes: Amplifying Internal Feature Representations for Language Model Interpretation](http://arxiv.org/abs/2503.02078v1), 2025-03-03

(97) [$\text{M}^3\text{HF}$: Multi-agent Reinforcement Learning from Multi-phase Human Feedback of Mixed Quality](http://arxiv.org/abs/2503.02077v1), 2025-03-03

(98) [CorrA: Leveraging Large Language Models for Dynamic Obstacle Avoidance of Autonomous Vehicles](http://arxiv.org/abs/2503.02076v1), 2025-03-03

(99) [Interactive Debugging and Steering of Multi-Agent AI Systems](http://dx.doi.org/10.1145/3706598.3713581), 2025-03-03

(100) [AI persuading AI vs AI persuading Humans: LLMs' Differential Effectiveness in Promoting Pro-Environmental Behavior](http://arxiv.org/abs/2503.02067v1), 2025-03-03
